editor/src/ui-tools/menu/menu.ts

```diff
@@ -136,7 +136,7 @@ export abstract class ItemMenu implements Menu {
       }
       const nextButton = buttons[nextIndex];
       this.focusButton(nextButton);
-      nextButton?.scrollIntoView({ behavior: 'smooth', block: 'nearest', inline: 'start' });
+      nextButton?.scrollIntoView({ behavior: 'auto', block: 'end', inline: 'end' });
```

Comment:
difference to master: `behavior: 'auto'` because instance is not available (maybe older ts version?)